resources/colony.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, amount, size, coordinates, color):
 
     def add_ants(self, amount_to_carry=1, step_size=3):
         for _ in range(self.amount):
-            self.ants.append(Ant(pheromone_status=0, 
+            self.ants.append(Ant(pheromone_status=-1, 
```

Comment:
I would suggest that the `Ant.pheromone_status` attribute should not receive its value from the `pheromone_status` parameter, as it does not make sense. Instead, the Ant.pheromone_status attribute should be initialized directly with -1, representing the initial state (coming from colony).

Instead of this:

```
class Ant:
    def __init__(self, pheromone_status, coordinates, amount_to_carry, step_size=1):
        ...
        self.pheromon_status = pheromone_status
```
        
You can do this:
```
class Ant:
    def __init__(self, coordinates, amount_to_carry, step_size=1):
        ...
        self.pheromon_status = -1
```

Reply:
Yes, I agree! I was thinking about it too, but I wasn't sure whether there was a reason behind it. I will change it.